tests/_support/AcceptanceTester.php

```diff
@@ -12,15 +15,226 @@
+				break;
+			case "rebuild":
+				$I->click(['xpath' => "//div[@id='toolbar-refresh']//button"]);
+				break;
+			case "trash":
+				$I->click(['xpath' => "//div[@id='toolbar-trash']//button"]);
+				break;
+			case "save":
+				$I->click(['xpath' => "//div[@id='toolbar-apply']//button"]);
+				break;
+			case "save & close":
+				$I->click(['xpath' => "//div[@id='toolbar-save']//button"]);
+				break;
+			case "save & new":
+				$I->click(['xpath' => "//div[@id='toolbar-save-new']//button"]);
+				break;
+			case "cancel":
+				$I->click(['xpath' => "//div[@id='toolbar-cancel']//button"]);
+				break;
+			case "options":
+				$I->click(['xpath' => "//div[@id='toolbar-options']//button"]);
+				break;
+			case "empty trash":
+				$I->click(['xpath' => "//div[@id='toolbar-delete']//button"]);
+				break;
+			case "Unblock":
+				$I->click(['xpath' => "//div[@id='toolbar-unblock']//button"]);
+				break;
+			case "delete":
+				$I->click(['xpath' => "//div[@id='toolbar-delete']//button"]);
+				break;
+			case "featured":
+				$I->click(['xpath' => "//div[@id='toolbar-featured']//button"]);
+				break;
+		}
+	}
+
+	/**
+	 * Function to select all the item in the Search results in Administrator List
+	 *
+	 * Note: We recommend use of checkAllResults function only after searchForItem to be sure you are selecting only the desired result set
+	 *
+	 * @return void
+	 */
+	public function checkAllResults()
+	{
+		$I = $this;
+		$this->comment("Selecting Checkall button");
+		$I->click(['xpath' => "//thead//input[@name='checkall-toggle' or @name='toggle']"]);
+	}
+
+	/**
+	 * Selects an option in a Chosen Selector based on its id
+	 *
+	 * @param   string  $selectId  The id of the <select> element
+	 * @param   string  $option    The text in the <option> to be selected in the chosen selector
+	 *
+	 * @return void
+	 */
+	public function selectOptionInChosenById($selectId, $option)
+	{
+		$chosenSelectID = $selectId . '_chzn';
+		$I = $this;
+		$this->comment("I open the $chosenSelectID chosen selector");
+		$I->click(['xpath' => "//div[@id='$chosenSelectID']/a/div/b"]);
+		$this->comment("I select $option");
+		$I->click(['xpath' => "//div[@id='$chosenSelectID']//li[text()='$option']"]);
+		$I->wait(1); // Gives time to chosen to close
+	}
+	/**
+	 * Function to Verify the Tabs on a Joomla! screen
+	 *
+	 * @param  Array  $expectedTabs   Expected Tabs on the Page
+	 * @param  Mixed  $tabsLocator    Locator for the Tabs in Edit View
+	 *
+	 * @return void
+	 */
+	/*public function verifyAvailableTabs($expectedTabs, $tabsLocator = ['xpath' => "//ul[@id='myTabTabs']/li/a"])
+	{
+		$I = $this;
+		$actualArrayOfTabs = $I->grabMultiple($tabsLocator);
+		$I->comment("Fetch the current list of Tabs in the edit view which is: " . implode(", ", $actualArrayOfTabs));
+		$url = $I->grabFromCurrentUrl();
+		$I->assertEquals($expectedTabs, $actualArrayOfTabs, "Tab Labels do not match on edit view of" . $url);
+		$I->comment('Verify the Tabs');
+	}*/
```

Comment:
Don't comment out code. We have source control. If you want to have a reminder, create an issue instead.